config/schema/og.schema.yml

```diff
@@ -153,3 +153,7 @@ condition.plugin.og_group_type:
       type: sequence
       sequence:
         type: string
+
+entity_reference_selection.og:default:
+  type: entity_reference_selection.default
+  label: 'OG entity selection handler settings'
```

Comment:
Schema change required due to https://www.drupal.org/node/2870971

Reply:
Let's try leaving this out, this hasn't proven to be problematic in the last 10 months so it was maybe a temporary hiccup that is no longer needed.